warp10/src/main/java/io/warp10/script/functions/SNAPSHOT.java

```diff
@@ -407,6 +416,90 @@ public static void addElement(SNAPSHOT snapshot, StringBuilder sb, Object o, boo
       } else if (o instanceof NamedWarpScriptFunction) {
         sb.append(o.toString());
         sb.append(" ");
+      } else if (o instanceof PGraphics) {
```

Comment:
This is subject to caution as this snapshot does not preserve the state of the PGraphics (selected font, current path, current attributes such as widtht, color, etc.)